common/gpu/amd/default.nix

```diff
@@ -0,0 +1,17 @@
+{ config, lib, pkgs, ... }:
+
+{
+  boot.initrd.kernelModules = [ "amdgpu" ];
```

Comment:
In nixpkgs we have `nixpkgs/nixos/modules/hardware/video/amdgpu.nix`.
I think we should extend it with it. We still could have a smaller module
here that enables this profile.
If you link the nixpkgs PR. I can have a look at it.

Reply:
Please clarify, you would like amdgpu set as a kernel module if it is set as an Xorg driver?

I could make a pull request like this for the file you mentioned:

```diff
  {
    config = mkIf (elem "amdgpu" config.services.xserver.videoDrivers) {
      boot.blacklistedKernelModules = [ "radeon" ];
+     boot.initrd.kernelModules = [ "amdgpu" ];
    };
  }

```

Reply:
I mean it could populate opengl.extraPackages. I don't have an opinion on wether it should enable hardware.opengl.

Reply:
The `amdgpu.nix` module has been removed in https://github.com/NixOS/nixpkgs/pull/111551#issuecomment-802082766.

I think we should continue work here.